nevergrad/functions/pyomo/core.py

```diff
@@ -141,9 +144,33 @@ def __init__(self, model: pyomo.Model) -> None:
         exp_tag = ",".join([n.name for n in self.all_objectives])
         exp_tag += "|" + ",".join([n.name for n in self.all_vars])
         exp_tag += "|" + ",".join([n.name for n in self.all_constraints])
-        self.register_initialization(name=exp_tag, model=self._model_instance)
+        self.register_initialization(model=self._model_instance)
+        #self.register_initialization(name=exp_tag, model=self._model_instance)
         self._descriptors.update(name=exp_tag)
 
+    def translate(self, budget):
+        """Stores in self._best_pyomo_val the best value obtained by a solver on the same instance for a given budget in a sequential optimization.
+        TODO: investigate what Pyomo does in the parallel case."""
```

Comment:
```suggestion
        """Offsets the loss by the value obtained by Pyomo's solver with a given budget.
        """
```
This docstring was providing implementation details that a user should not be aware of